src/test/java/net/sf/jabref/importer/fileformat/MedlinePlainImporterTest.java

```diff
@@ -0,0 +1,122 @@
+                "IsiImporterTestInspec.isi", "IsiImporterTestWOS.isi", "IsiImporterTestMedline.isi",
+                "MedlinePlainImporterTestEmpty.txt");
+
+        //Test with txt Files which are not in recognized Format
+        for (String str : list) {
+            try (InputStream is = MedlinePlainImporter.class.getResourceAsStream(str)) {
+                Assert.assertFalse(importer.isRecognizedFormat(is));
+            }
+        }
+
+        list = Arrays.asList("MedlinePlainImporterTest.txt", "MedlinePlainImporterTest1.txt",
+                "MedlinePlainImporterTest2.txt", "MedlinePlainImporterTest3.txt", "MedlinePlainImporterTest4.txt",
+                "MedlinePlainImporterTest5.txt");
+
+        //Test with txt File which is in recognized Format
+        for (String str : list) {
+            try (InputStream is = MedlinePlainImporter.class.getResourceAsStream(str)) {
+                Assert.assertTrue(importer.isRecognizedFormat(is));
+            }
+        }
+    }
+
+    @Test
+    public void testImportMultipleEntriesInSingleFile() throws Exception {
+        try (InputStream is = MedlinePlainImporter.class.getResourceAsStream("MedlinePlainImporterTest.txt")) {
+            //create entries list from the txt file
+            List<BibEntry> entries = importer.importEntries(is, new OutputPrinterToNull());
+            Assert.assertEquals(7, entries.size());
+
+            //create Bibtexentry from first/second object in the entry list
+            //and test with getField
+            BibEntry test1 = entries.get(0);
+            Assert.assertNull(test1.getField("month"));
+            Assert.assertEquals("Long, Vicky and Marland, Hilary", test1.getField("author"));
+            Assert.assertEquals(
+                    "From danger and motherhood to health and beauty: health advice for the factory girl in early twentieth-century Britain.",
+                    test1.getField("title"));
+            BibEntry test2 = entries.get(1);
+            Assert.assertEquals("06", test2.getField("month"));
+            Assert.assertNull(test2.getField("author"));
+            Assert.assertNull(test2.getField("title"));
+        }
+    }
+
+    @Test
+    public void testEmptyFileImport() throws Exception {
+        try (InputStream is = MedlinePlainImporter.class.getResourceAsStream("MedlinePlainImporterTestEmpty.txt")) {
+            List<BibEntry> emptyEntries = importer.importEntries(is, new OutputPrinterToNull());
+            Assert.assertEquals(0, emptyEntries.size());
+        }
+    }
+
+    @Test
+    public void testImportSingleEntriesInSingleFiles() throws IOException {
+        for (int n = 1; n <= 5; n++) {
+            String medlineFile = "MedlinePlainImporterTest" + n + ".txt";
+            String bibtexFile = "MedlinePlainImporterTestBib" + n + ".bib";
+            assertImportOfMedlineFileEqualsBibtexFile(medlineFile, bibtexFile);

```

Comment:
Please directly test that the imported entries are correct (as it is done above). The test as it stands could also fail because the bib-file import has a bug.